### PR TITLE
Added CloudFlareService.get_ssl_verification_info()

### PR DESCRIPTION
**Pull request by ekoval**
Closes: #33

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: tests/test_integration.py [hunk 1]
```diff
@@ -136,6 +154,9 @@ def setUp(self):
         self.cfh = CloudFlareHostService()
         self.user = self.cfh.user_lookup(email=TEST_USER['email'])
 
+    def tearDown(self):
+        tearDownModule()
```
Comment (RayeN) [inline tests/test_integration.py]: Why do we need this?

File: pycloudflare/services.py [hunk 2]
```diff
@@ -105,6 +105,9 @@ def purge_cache(self, zone_id, files=None, tags=None):
             data['purge_everything'] = True
         return self.delete('zones/%s/purge_cache' % zone_id, json=data)
 
+    def get_ssl_verification_info(self, zone_id):
+        return self.get('/zones/%s/ssl/verification' % zone_id)
```
Comment (RayeN) [inline pycloudflare/services.py]: we don't use leading slash in other methods.

File: tests/test_integration.py (hunk 1 above)
Comment (RayeN) [inline tests/test_integration.py]: I think it's called automagically.

**Comment by RayeN** [comment]
I think we need appropriate method in Zone model, because raw service method isn't convenient to use (we don't have zone_id).

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## DEV
+
+* Add `models.Zone.get_ssl_verification_info` and corresponding service
+  support.
```
Comment (RayeN) [inline CHANGELOG.md]: I would add separate log entries for service method and model's method.  But that's not critical.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Pull request merged**